### DISCUSSION TITLE
Failed to run binary file rke2 to join cluster "--token is required"

### DISCUSSION THREAD

**Discussion by khawschuy** [rancher/rke2#7446] [Q&A] [answered]
I have followed all the configuration steps.
![image](https://github.com/user-attachments/assets/64f2d8b3-b5bc-40d8-b58e-9c0b1264b360)
When I run the command
`systemctl start rke2-agent.service`
I encounter an error message:
![{1FAE6243-ECCE-4E55-AECE-B04C66ACFACE}](https://github.com/user-attachments/assets/b6497e48-cedc-442e-8896-573f7ee117e5)

Then, I try running
`systemctl status rke2-agent.service`
And the error message is
![{44840FC2-DC97-4954-829E-FE54DA6CF87D}](https://github.com/user-attachments/assets/064fc188-11b6-47de-aaa9-093aa44b2220)

I also tried running the binary file directly and encountered a similar error:
![{3ED564DA-5DEA-4148-A9F3-67DC2959BBE9}](https://github.com/user-attachments/assets/4323c987-e860-4972-afea-26125beeddad)

I can join 2 master node to cluster so, I think that could be agent's binary error
![{72DE2FE3-2C03-44F4-976A-1901262B998C}](https://github.com/user-attachments/assets/6f30e363-768f-441f-9aa2-0a8cb45ec7ef)

**Comment by brandond** [comment, marked as answer]
The file name should be config.yaml, not conf.yaml.
